packages/runner-docker/index.mjs

```diff
@@ -0,0 +1,137 @@
+import { execFile } from 'node:child_process';
+import { join, resolve } from 'node:path';
+import { readFile } from 'node:fs/promises';
+import os from 'node:os';
+import ac from '@expressjs/perf-autocannon';
+
+export async function startServer (opts = {}) {
+  return new Promise((resolve, reject) => {
+    if (opts?.signal.aborted) {
+      return reject(opts.signal.reason);
+    }
+
+    const cp = execFile(
+      join(import.meta.dirname, 'scripts', 'run.sh'),
+      [
+        // TODO: convert this to json, there is much more to pass?
```

Comment:
I think I changed course on this todo. We need to put more of the configuration into the test so that can be loaded with just the one flag. So this should be the entire list of input options necessary.